policies/queries/ecr/repositories_unused.sql

```diff
@@ -0,0 +1,14 @@
+insert into aws_policy_results
+with image as (select distinct repository_cq_id from aws_ecr_repository_images)
+select :'execution_time'                           as execution_time,
+       :'framework'                                as framework,
+       :'check_id'                                 as check_id,
+       'Unused ECR repositories should be removed' as title,
+       repository.account_id,
+       repository.arn                              as resource_id,
+       case
+           when image.repository_cq_id is null then 'fail'
+           else 'pass'
+           end                                     as status
+from aws_ecr_repositories repository
+         left join image on image.repository_cq_id = repository.cq_id
```

Comment:
What happens if `aws_ecr_repository_images` wasn't fetched?

Reply:
I suppose it'll fail on the query, as t should?